File: 프로그래머스/Lv.0/외계행성의 나이/120834.js
function solution(age) {
    let answer = '';
    
    const alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j'];
    while (age > 0) {
        answer = alphabet[age % 10] + answer;
        age = parseInt(age / 10);
    }
    
    return answer;
}

File: 프로그래머스/Lv.0/외계행성의 나이/120834.java
class Solution {
    final static String[] alphabet = {"a", "b", "c", "d", "e", "f", "g", "h", "i", "j"};
    
    public String solution(int age) {
        String answer = "";
        
        for (String a : String.valueOf(age).split("")) {
            answer += alphabet[Integer.parseInt(a)];
        }
        
        return answer;
    }
}
